python/sedona/sql/functions.py

```diff
@@ -0,0 +1,126 @@
+import inspect
```

Comment:
Apache File header please

Reply:
I didn't realize that it is not added automatically